src/components/InlineLoading/InlineLoading.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import Loading from '../Loading';
+
+export default class InlineLoading extends React.Component {
+  static propTypes = {
+    className: PropTypes.string,
+    success: PropTypes.bool,
+    description: PropTypes.string,
+    onSuccess: PropTypes.func,
+  };
+
+  static defaultProps = {
+    success: false,
+  };
+
+  render() {
+    const { className, success, description, onSuccess, ...other } = this.props;
+
+    const loadingClasses = classNames('bx--inline-loading', className);
+
+    const getLoading = () => {
+      if (success) {
+        setTimeout(() => {
+          if (onSuccess) {
+            onSuccess();
+          }
+        }, 1500);
```

Comment:
Can we make this number as a prop?

Reply:
Yup! I wasn't sure if we would want to the success time to be consistent or not, but I'm happy for it to be a prop.

Reply:
Great - I think we can make `1500` the default. Thanks @loganmccaul!

Reply:
Did you want to make this a prop before merging @loganmccaul ?

Reply:
Oh, must've never pushed. My fault 😄